cypress/e2e/po/detail/provisioning.cattle.io.cluster/cluster-detail.po.ts

```diff
@@ -21,11 +26,33 @@ export default abstract class ClusterManagerDetailPagePo extends PagePo {
     return this.self().find('.primaryheader h1');
   }
 
+  logsContainer() {
+    return new ClusterProvisioningLogPo(this.self()).logsContainer();
+  }
+
   kubectlCommandForImported() {
     return this.self().get('code').contains('--insecure');
   }
 
   machinePoolsList() {
     return new MachinePoolsListPo(this.self().find('[data-testid="sortable-table-list-container"]'));
   }
+
+  conditionsList() {
+    return new ClusterConditionsListPo(this.self().find('.sortable-table'));
+  }
+
+  referredToList() {
+    return new ClusterReferredToListPo(this.self().find('[data-testid="sortable-table-list-container"]'));
+  }
+
+  downloadYamlButton() {
+    return new ResourceTablePo(this.self()).downloadYamlButton();
+  }
```

Comment:
this button is specific to machine pool, let's move it to `MachinePoolsListPo`

Reply:
@izaac just this one left then we're good to go

Reply:
Github is acting for me this morning, I've seen your comment yesterday here in the Web. But I just got an email 10 minutes ago that I had this as a new comment so I'm not so sure if this is new.

Anyhow, I updated it yesterday.

Reply:
Oh weird, the message I sent yesterday disappeared on my end and I didn't see your changes so I sent the message again this morning but okay I'll take a look